envsec/envsec.go

```diff
@@ -20,12 +22,20 @@ type EnvId struct {
 	EnvName string
 }
 
-func NewEnvId(projectId string, orgId string, envName string) (EnvId, error) {
-	if projectId == "" {
+func NewEnvId(orgId string, envName string) (EnvId, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return EnvId{}, errors.New("Failed to get current workding directory")
+	}
+	projectId, err := jetcloud.ProjectID(wd)
+	if err != nil {
+		return EnvId{}, errors.New("Failed to retrieve projectID")
+	}
+	if projectId.String() == "" {
 		return EnvId{}, errors.New("ProjectId can not be empty")
 	}
 	return EnvId{
-		ProjectId: projectId,
+		ProjectId: projectId.String(),
```

Comment:
I don't think we should make these changes to NewEnvId. They make this project not work as an open source project anymore.